mk/lint.mk

```diff
@@ -0,0 +1,15 @@
+.PHONY: install-pre-commit, install-golangci-lint, install-linter, lint, lint-all
+
+install-pre-commit:
+	pip install pre-commit
```

Comment:
Consider running `pre-commit install-hooks` here too, so it will start taking effect.